### PR TITLE
Update usage dashboard docs [APP-198]

### PR DESCRIPTION
**Pull request by katthartic**

### CODE REVIEW

File: docs/reference/admin/usage-dashboard.md [hunk 1]
```diff
@@ -14,36 +14,45 @@ keywords:
...
-In addition, the usage dashboard will show you:
+#### Overall Metrics
 
-- The date range of the current billing period.
-- The % of the current billing period that has already passed.
-- The last date the data was updated.
-- The approximate invoice total for the current billing period.
+Based on the selected filters, you'll see:
+- **Total Runs**: The cumulative count of the Fleet runs.
+- **Total Runtime**: The total duration of the Fleet runs.
```
Comment (blakeburch) [inline docs/reference/admin/usage-dashboard.md:32]: The cumulative billable runtime...

File: docs/reference/admin/usage-dashboard.md [hunk 2]
```diff
@@ -14,36 +14,45 @@ keywords:
...
+- **Average Runtime**: The average duration of the Fleet runs.
```
Comment (blakeburch) [inline docs/reference/admin/usage-dashboard.md:33]: The average billable runtime...

File: docs/reference/admin/usage-dashboard.md (hunk 1 above)
Comment (blakeburch) [reply docs/reference/admin/usage-dashboard.md:32]: Can't say duration. That's a different metric.

File: docs/reference/admin/usage-dashboard.md (hunk 2 above)
Comment (blakeburch) [reply docs/reference/admin/usage-dashboard.md:33]: Can't say duration. That's a different metric.

**Review by blakeburch — COMMENTED** [review]
Minor syntax changes needed for clarity.

**Comment by katthartic** [comment]
@blakeburch Thanks for the fast review! I've addressed your feedback.

**Review by blakeburch — APPROVED** [review]

**Pull request merged**